src/TunnelEncapsulation.h

```diff
@@ -46,6 +46,7 @@ class EncapsulatingConn {
           src_port(0),
           dst_port(0),
           proto(TRANSPORT_UNKNOWN),
+          proto_id(255),
```

Comment:
We used 256 earlier, more reason to alias it to a constant signifying "unknown".

Reply:
Added a constant to IP.h, used it in Conn.h and TunnelEncapsulation.h.